.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 6.8.0
```

Comment:
FYI You don't need to bumpversion because you're not changing the python API (anything in `routes/` or `metamist/`).

Reply:
Thanks for clarifying, removed the bumpversion changes